packages/cli/src/commands/governance/approve.test.ts

```diff
@@ -128,4 +151,102 @@ testWithAnvil('governance:approve cmd', (web3: Web3) => {
     `)
     expect(writeMock.mock.calls).toMatchInlineSnapshot(`[]`)
   })
+
+  describe('approve succeeds if stage is "Referendum or Execution" or "Approval"', () => {
+    test('can be approved if version >= 3 (default)', async () => {
+      const logMock = jest.spyOn(console, 'log')
+
+      await governance
+        .propose([], 'https://example.com')
+        .sendAndWaitForReceipt({ value: minDeposit })
+
+      const approver = await governance.getApprover()
+      await sendRawTx('anvil_impersonateAccount', [approver])
+      await sendRawTx('anvil_setBalance', [approver, '0x10000000000000000000'])
```

Comment:
please create a helper function for this as well

it's actually good to know there's such possibility (instead of trying to transfer funds to an address) 👍